### DISCUSSION TITLE
My device does not detect any bluetooth devices

### DISCUSSION THREAD

**Discussion by Dnxx2** [blueman-project/blueman#2441] [Q&A]
blueman: 2.3.5
BlueZ:5.64-0ubuntu1.1
Distribution: Linux Mint 21.3
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Ever since I installed linux mint on my Ideapad 1i I wanted to connect my bluetooth headphones to my laptop but I never could I tried every forum of the internet in order to troubleshoot my problem and it does not work. Is my case any different to everybody else?, and if it is, how could i solve it?

**Comment by cschramm** [comment]
Check `dmesg` for driver issues (on search or on boot)
